### PR TITLE
R spec category and product model

### PR DESCRIPTION
**Pull request by namtm-1272**
## Related Tickets
- [#53214](https://edu-redmine.sun-asterisk.vn/issues/53214)

## WHAT (optional)
- Write Rspec for category and product model

## HOW
- create factories for FactoryBot data
- write model category_spec and product_spec

## WHY (optional)
- Cần test trước khi đưa ra sản phẩm

## Evidence (Screenshot or Video)
Screenshot from 2022-09-12 09-03-45

### CODE REVIEW

**Comment by namtm-1272** [comment]
ready

**Comment by namtm-1272** [comment]
ready

File: spec/models/product_spec.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    end
+  end
+
+  describe "scope" do
+    let!(:product1){FactoryBot.create :product, name: "salad", description: "sfsdfsfj", unit_price: "89", quantity: "10"}
+    let!(:product2){FactoryBot.create :product, name: "beef", description: "sfsdfsfj", unit_price: "50", quantity: "12" }
+    let!(:product3){FactoryBot.create :product, name: "wine", description: "sfsdfsfj", unit_price: "60", quantity: "15" }
+
+    it "asc_name" do
+      expect(Product.asc_name.pluck(:id)).to eq([product2.id, product1.id, product3.id])
```
Comment (huongnt-2545) [inline spec/models/product_spec.rb:34]: dùng matcher `should, should_not` thay vì `expect` nhé
sửa tương tự bên dưới

**Comment by namtm-1272** [comment]
ready

**Pull request merged**